self-service-kiosk/src/app/config/order-config.js

```diff
@@ -71,9 +73,19 @@ export default {
             let order = await extractBody(`http://localhost:${diningPort}/dining/tableOrders/${orderID}`, option);
             return order['lines'];
         },
-        removeItemToOrder: async (orderID) => {
 
+        removeItemToOrder: async ({orderID, menuItem}) => {
+            let diningPort = 9500;
```

Comment:
Shouldn't we change this setting to a configuration file?

Reply:
yes